### ISSUE TITLE
Windows Native Errors

### ISSUE THREAD

**Issue by 78Alpha** [VoltaML/voltaML-fast-stable-diffusion#47]
### Describe the bug

AITemplate has an issue compiling models due to it complaining about the Make file being missing. Manually editing the make file for windows specific directory structures does not work (has not in the past). Module Aitemplate can't be imported unless built manually, and even then, like TensorRT, it fails to work on windows out of the box.

### Reproduction

1. Install the repo as you would on any other system
2. Manually build AItemplate
3. Install AITemplate Wheel (due to repo not being able to import Pypi AITemplate)
4. Attempt to accelerate model
5. Receive error about missing makefile directory

### Expected behavior

Honestly, I expected this, most speedups are Linux only (WSL or otherwise).

### Branch

Main

### System Info

Python: 3.10
OS: Windows 11
Repo: https://github.com/VoltaML/voltaML-fast-stable-diffusion/commit/6c82d0558d38156d4d9d6d5f61d62d91c52f62e1
GPU: RTX 3090
RAM: 48 GB

### Additional context

FileNotFoundError: [Errno 2] No such file or directory: "'data\\aitemplate\\Linaqruf--anything-v3.0__512x512x1\\profiler'\\Makefile"

### Validations

- [X] Read the [docs](https://voltaml.github.io/voltaML-fast-stable-diffusion/).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

**Comment by Stax124** [comment]
Can you please start it like this: `python main.py --log-level DEGUB | tee debug.log` in powershell? CMD doesnt support `tee` I think. Then send me the log on Discord (or Upload to other site and include link here).

Did you clone the AIT repo with `--recursive`? It needs all the deps to work correctly.

**Comment by 78Alpha** [comment]
Remade the environment with `--recursive` just to make sure and it ran into the same issue.

[https://pastebin.com/Xs1UC46m](https://pastebin.com/Xs1UC46m)

If I remember correctly from my last run at AITemplate, it has the error because it adds `'` into the directory path and Windows treats `C:\Makefile` as a directory. Changing it to makefile.txt lets it write paths to the makefile, but they are Linux style paths that windows doesn't like. I once marked it as read only (prevent it from filling it with junk again) and changed all the paths to their windows equivalent by hand but it still had issues (foggy at this part).

And I must add, this is Windows Native, no WSL or Virtualization. Comes up https://github.com/facebookincubator/AITemplate/issues/83 but that is about building. May just be hard coded or Make on windows is different.

**Comment by Stax124** [comment]
We tried that as well with one guy from our community yesterday.
Even if you get through the preparation process, you get errored out with the compilation as it requires a program called Make, a Linux-only program that is also dependent on external stuff. I tried to use make.exe from the GNUWin32 project, but even then it throws out some errors - that are unfortunately in the build process.

I will create a guide (soon) on how to install it locally on Windows - but with WSL - so you do not need to use docker.
